src/allmydata/test/common_util.py

```diff
@@ -203,6 +203,11 @@ def flip_one_bit(s, offset=0, size=None):
 class ReallyEqualMixin(object):
     def failUnlessReallyEqual(self, a, b, msg=None):
         self.assertEqual(a, b, msg)
+        # Make sure unicode strings are a consistent type:
```

Comment:
If they're not a consistent type then how would we get here, since `"foo" != b"foo"`? Doesn't the `assertEqual` just before fail in that case?

Reply:
It's because it's the Future newstr (backported Unicode type) vs. Python 2 native unicode type. So they're equal, and _logically_ the same type, but have different types in practice. I'll update the comment.